staging/src/k8s.io/client-go/kubernetes_test/fake_client_test.go

```diff
@@ -0,0 +1,77 @@
+package kubernetes
+
+import (
+	"context"
+	"testing"
+
+	corev1 "k8s.io/kubernetes/pkg/apis/core/v1"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+// this test proves that the kube fake client does not return GVKs
```

Comment:
add a TODO comment, so readers know this is not desired, and should be fixed?

Reply:
> add a TODO comment, so readers know this is not desired, and should be fixed?

sure.  I will clarify.